code/modules/mob/living/carbon/human/species_types/abductors.dm

```diff
@@ -17,3 +16,8 @@
 	. = ..()
 	var/datum/atom_hud/abductor_hud = GLOB.huds[DATA_HUD_ABDUCTOR]
 	abductor_hud.remove_hud_from(C)
+
+/datum/species/abductor/get_species_organs()
+	var/list/organs = ..()
+	organs[ORGAN_SLOT_TONGUE] = /obj/item/organ/tongue/abductor
+	return organs
```

Comment:
Again, code was better before as single variable checks are better than dumping it all into a list